common/script/ops/reset.js

```diff
@@ -1,35 +1,22 @@
-import _ from 'lodash';
+import resetGear from './resetGear';
+import i18n from '../i18n';
 
-module.exports = function(user, req, cb) {
-  var gear;
-  user.habits = [];
-  user.dailys = [];
-  user.todos = [];
-  user.rewards = [];
+module.exports = function reset (user) {
+  user.tasksOrder.habits = [];
```

Comment:
Otherwise you can pass tasks to `reset` and remove the ids from `tasksOrder` here in the shared code. Then you return also the removed ids and use them to executed the server query